src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+abstract class Machine {

```

Comment:
The class `Machine` should be public to be accessible from other packages, especially since it's an abstract class and is meant to be extended. Remember to use the appropriate access modifiers to follow the encapsulation principle.